autoware_map_msgs/msg/LaneletMapTileMetaData.msg

```diff
@@ -0,0 +1,5 @@
+# Metadata of lanelet map tile
+
+string mgrs_code
+float64 origin_lat
+float64 origin_lon
```

Comment:
Is there any reason why you did not include the `int32 tile_id` here?

Reply:
I just made this similar with point cloud metadata files but this with this way, it look better. Updated.

Reply:
@yukkysaito Why did you include `int32 tile_id` here?


Now, the structure of lanelet2 is :
LaneletMapTileMetaData.msg
```
int32 tile_id
float64 min_x
float64 max_x
float64 min_y
float64 max_y
```

However, the structure of  PointCloud is:
PointCloudMapCellMetaData.msg
```
float32 min_x
float32 min_y
float32 max_x
float32 max_y
```
PointCloudMapCellMetaDataWithID.msg
```
string cell_id
PointCloudMapCellMetaData metadata
```

I am fine with either structure, include `tile_id` or not.
But I would like to unify the structure of lanelet2 and PointCloud.

Reply:
Are there any case where we use PointCloudMapCellMetaData without an id?
If that's so we can keep them as a separate message, but if they are meant to be used together most of the time, then I think we can put them together as a single message type.

Reply:
>But I would like to unify the structure of lanelet2 and PointCloud.

I agree :+1: 

>Are there any case where we use PointCloudMapCellMetaData without an id?

Would @kminoda  know the answer to this?

Reply:
Currently, there is no code using PointCloudMapCellMetaData without an id.
If we need to change PointCloudMapCellMetaData, we can do so without much effort.
If it's not a design anticipating something for the future, I think it's fine to modify PointCloudMapCellMetaData.


@kminoda I'd like to hear your opinion

Reply:
I think it's ok to gather those information in one msg file (both `cell_id` and `min/max_x/y`. Not sure about the reason why I did so, but there is no specific reason for it as far as I can remember.

Reply:
@StepTurtle @mitsudome-r @yukkysaito @kminoda 
Thank you all!
I will make a PR to modify `PointCloudMapCellMetaData.msg` as below and delete `PointCloudMapCellMetaDataWithID.msg`.

PointCloudMapCellMetaData.msg
```
string cell_id
float32 min_x
float32 min_y
float32 max_x
float32 max_y
```

Reply:
LGTM :+1: